### ISSUE TITLE
Clear data and settings of Scopes

### ISSUE THREAD

**Issue by YugiFanGX** [ubports/ubuntu-touch#905]
Ubuntu Touch Settings should have an app menu for managing permissions, clearing cache, data and other settings

**Comment by advocatux** [comment]
Have you tried UT Tweak tool? (https://open-store.io/app/ut-tweak-tool.sverzegnassi)

If after trying that app you still miss any function, probably it would be easier to add any kind of feature to that tool than implement them in System Settings from scratch.

**Comment by giiba** [comment]
I agree with YugiFanGX, this is a basic function that shouldn't require an additional app.

But at least the function is available somewhere.
